### PR TITLE
feat(nginx-logs): some improvements

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-676

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> You are going to thank you later if it contains tests

File: libs/shared/ui/src/lib/components/inputs/input-checkbox/input-checkbox.tsx [hunk 1]
```diff
@@ -60,15 +60,15 @@ export function InputCheckbox(props: InputCheckboxProps) {
         onChange={(e) => inputChange(e.currentTarget.checked, e)}
         className={`input-checkbox relative font-icons w-0 h-0 appearance-none before:absolute before:flex before:justify-center before:items-center before:text-white before:top-0 before:left-0 before:-translate-y-1/2 before:rounded-sm before:bg-white dark:before:bg-element-light-darker-100 ${bigClasses} ${
           disabled
-            ? 'before:border-element-light-lighter-500'
+            ? 'before:border-element-light-lighter-500 dark:before:border-element-light-lighter-700'
```
Comment (ctjhoa) [inline libs/shared/ui/src/lib/components/inputs/input-checkbox/input-checkbox.tsx:63]: Its gonna conflict with color name change no?

**Review by ctjhoa — APPROVED** [review]

File: libs/shared/ui/src/lib/components/inputs/input-checkbox/input-checkbox.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/shared/ui/src/lib/components/inputs/input-checkbox/input-checkbox.tsx:63]: ATM, I didn't touch token with `element` colors, I'll add it soon 👍

**Pull request merged**